### ISSUE TITLE
[3.0.7] Purpur slabs uncraftable/request new recipe for Quark's Weather Sensor/request alternative to Quark's Weather Sensor

### ISSUE THREAD

**Issue by dalsio** [DarkPacks/SevTech-Ages#2551]
## Issue / Bug
In my frustrations with my BWM windmill always breaking, I sought a way to sense the weather and emit a redstone signal. Upon searching, the only one available to me at stage 4 is the Weather Sensor from quark which requires purpur slabs which I notice cannot be crafted. Not only that, but they can't be converted from slabs back to blocks. Is this intentional? If so, is there an alternative to Quark's Weather Sensors? If not, can there be one, or can it have a different recipe?

## Expected Behavior
I can craft purpur slabs from blocks OR I can make Quark's Weather Sensor without purpur slabs OR I can make a different weather sensor with some other material.

## Possible Solution
Config/script work

## Steps to Reproduce (for bugs)
1. Open JEI
2. Search for a weather/rain sensor
2. Check the recipe for Quark's Weather sensor
3. Check the recipe for Purpur slabs

## Context
It means I have to go all the way back to the End to harvest specifically purpur slabs for any weather sensors I might want now or in the future, if the purpur fortress things even have any slabs in them OR continue to suffer.

## Client Information
* Modpack Version: 3.0.7
* Java Version: 1.8.0.171
* Launcher Used: Twitch, modified to use the latest java
* Memory Allocated: 4gb
* Server/LAN/Single Player: Single
* Resourcepack Enabled?: No
* Optifine Installed?: No

**Comment by Stifling-Bossness** [comment]
Purpur slab recipes are available at stage 5, so this is probably just a staging issue.

**Comment by dalsio** [comment]
Yea, I guess then either they need to be enabled stage 4 (which makes more sense) or the weather sensor disabled until stage 5. I found (through much digging) some alternative sensors to this one, but only after I harvested some slabs and crafted it anyway. Definitely a low priority but also an easy fix.

**Comment by Jandakast** [comment]
If you check the BWM book, under Blocks -> redstone devices there is a Wind Chime that does exactly this and is craftable in age 1.

It is intentional for the player not to be able to craft the purpur blocks until age 5. They must explore and gather the blocks from the end cities until they unlock Age 5
